tests/testdata/keras_tuner_mnist_example.py

```diff
@@ -0,0 +1,48 @@
+from tensorflow import keras
+from tensorflow.keras import layers
+
+
+from kerastuner.tuners import RandomSearch
+from kerastuner.engine.hypermodel import HyperModel
+from kerastuner.engine.hyperparameters import HyperParameters
+
+
+(x, y), (val_x, val_y) = keras.datasets.mnist.load_data()
```

Comment:
I would recommend using a CIFAR10 convnet, to justify the use of the V100_4X config. A state-of-the-art CIFAR10 convnet should be achievable via random search in a reasonable amount of time.

I would also recommend starting the model with image data augmentation layers and including optional dropout in the search (before the classification layer)

Reply:
Done.